src/test/java/de/neuefische/linkedlist/AnimalListTest.java

```diff
@@ -0,0 +1,297 @@
+package de.neuefische.linkedlist;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AnimalListTest {
+
+    @Test
+    public void testListBuilder() {
+        //GIVEN
+        Animal pferd = new Animal("Pferd");
+        Animal affe = new Animal("Affe");
+        Animal elefant = new Animal("Elefant");
+        AnimalList testList = new AnimalList(pferd);
+        //WHEN
+        String expected = "Pferd";
+        //THEN
+        String actual = testList.toString();
+        assertEquals(expected, actual);
```

Comment:
```suggestion
        //GIVEN
        Animal pferd = new Animal("Pferd");
        Animal affe = new Animal("Affe");
        Animal elefant = new Animal("Elefant");
        AnimalList testList = new AnimalList(pferd);
        
        //WHEN
        String actual = testList.toString();
        
        //THEN
        String expected = "Pferd";
        assertEquals(expected, actual);
```
Try not to mix the different blocks